tests/imodels-clients-tests/src/integration/management/ChangesetGroupOperations.test.ts

```diff
@@ -0,0 +1,87 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { AuthorizationCallback, GetChangesetGroupListParams, GetSingleChangesetGroupParams, IModelsClient, IModelsClientOptions, IModelsErrorCode } from "@itwin/imodels-client-management";
+import { ReusableIModelMetadata, ReusableTestIModelProvider, TestAuthorizationProvider, TestUtilTypes, assertChangesetGroup, assertCollection, assertError } from "@itwin/imodels-client-test-utils";
+
+import { getTestDIContainer } from "../common";
+
+describe("[Management] ChangesetGroupOperations", () => {
+  let iModelsClient: IModelsClient;
+  let authorization: AuthorizationCallback;
+  let testIModel: ReusableIModelMetadata;
+
+  before(async () => {
+    const container = getTestDIContainer();
+
+    const iModelsClientOptions = container.get<IModelsClientOptions>(TestUtilTypes.IModelsClientOptions);
+    iModelsClient = new IModelsClient(iModelsClientOptions);
+
+    const authorizationProvider = container.get(TestAuthorizationProvider);
+    authorization = authorizationProvider.getAdmin1Authorization();
+
+    const reusableTestIModelProvider = container.get(ReusableTestIModelProvider);
+    testIModel = await reusableTestIModelProvider.getOrCreate();
+  });
+
+  it("should return all items when querying collection", async () => {
```

Comment:
let's add a test that not only asserts collection length but also validates at least one of the entities returned in a collection, like this test for example
https://github.com/iTwin/imodels-clients/blob/3a8ce20e9ffc36e4f1192e123d39507d3b2c0d3e/tests/imodels-clients-tests/src/integration/management/NamedVersionOperations.test.ts#L174

Reply:
Added